### PR TITLE
Fix inconsistent alerts by using the markdown notation - part 2

### PR DESCRIPTION
**Pull request by akordowski**
### Why:

As described in the issue #35195 the display of alerts is inconsistent across the documentation.

As the original PR #35216 exceeds the linter limit of 300 files, this PR is part 2 of the changes. It provides changes to alerts across the documentation according to the docs [guidelines](https://docs.github.com/en/contributing/writing-for-github-docs/using-markdown-and-liquid-in-github-docs#alerts) using the Markdown notation.

### What's being changed (if available, include any code snippets, screenshots, or gifs):

Alerts for `note`, `tip`, `important` and `warning` have been changed and unified from the liquid notation (`{% note %}`) to the Markdown notation (`> [!NOTE]`) according to the docs guidelines.

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline (this link will be available after opening the PR).

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

### CODE REVIEW

**Comment by nguyenalex836** [comment]
@akordowski Reviewing this now! 💛

File: content/code-security/code-scanning/integrating-with-code-scanning/sarif-support-for-code-scanning.md [hunk 1]
```diff
@@ -313,86 +310,6 @@ This SARIF output file has example values to show the minimum required propertie
 }
 ```
 
-### Relative URI Guidance for SARIF Producers
```
Comment (nguyenalex836) [inline content/code-security/code-scanning/integrating-with-code-scanning/sarif-support-for-code-scanning.md:316]: @akordowski This content was introduced via another recently merged PR - https://github.com/github/docs/pull/35043

Can we ensure this content remains intact? 💛

Comment (akordowski) [reply content/code-security/code-scanning/integrating-with-code-scanning/sarif-support-for-code-scanning.md]: I restored the content and updated the branch.

Comment (nguyenalex836) [reply content/code-security/code-scanning/integrating-with-code-scanning/sarif-support-for-code-scanning.md]: Thank you so much! 💛

**Review by nguyenalex836 — APPROVED** [review]

**Comment by nguyenalex836** [comment]
@akordowski Part 2 approved! ✨ 

I know we're only halfway through this PR series, but I wanted to relay thanks from the team for this undertaking 💛 I'll have this merged shortly.

**Comment by akordowski** [comment]
@nguyenalex836 

> I wanted to relay thanks from the team for this undertaking

Thank you very much! ☺️ It's my pleasure to give something back.

> I'll have this merged shortly.

No rush! Thank you for your effort to review this big chunks.

**Pull request merged**